config/routes.rb

```diff
@@ -66,6 +66,16 @@
       put 'admin_update'
     end
   end
+  
+  resources :departments, :path => 'org/admin/department', only: [] do
```

Comment:
you can then remove this and add it under the `org_admin` namespace as a single line: 
```ruby
namespace :org_admin do
  ...
  resources :departments
  ...
end
```